src/shogun/machine/gp/SingleSparseInference.cpp

```diff
@@ -112,8 +111,10 @@ class SingleSparseInferenceCostFunction: public FirstOrderBoundConstraintsCostFu
         void init()
 	{
 		m_obj=NULL;
-		SG_ADD((CSGObject **)&m_obj, "CSigleSparseInference__m_obj",
-			"m_obj in AdamUpdater", MS_NOT_AVAILABLE);
+		//The existing implementation in CSGObject::get_parameter_incremental_hash()
```

Comment:
Ah this is annoying. I guess we can fix that with the hash in the new any.
@sanuj @lisitsyn  ?